Sources/App/Scenes/WindowSizeObserver.swift

```diff
@@ -23,6 +24,11 @@ final class WindowSizeObserver: NSObject {
         }
         #endif
     }
+
+    public func stopObserving() {
+        observation?.invalidate()
```

Comment:
couldn't this go in deinit instead? i guess being explicit is fine here though

Reply:
It could, but since we call "start" I think it's easier to follow the implementation if we also have an explicit "stop"